build.gradle.kts

```diff
@@ -16,23 +16,23 @@ dependencies {
   implementation("org.springframework.boot:spring-boot-starter-data-jpa")
   implementation("org.apache.commons:commons-text:1.9")
   implementation("io.swagger:swagger-annotations:1.6.6")
-  implementation("org.springdoc:springdoc-openapi-ui:1.6.8")
-  implementation("org.springdoc:springdoc-openapi-kotlin:1.6.8")
-  implementation("org.springdoc:springdoc-openapi-data-rest:1.6.8")
+  implementation("org.springdoc:springdoc-openapi-ui:1.6.11")
+  implementation("org.springdoc:springdoc-openapi-kotlin:1.6.11")
+  implementation("org.springdoc:springdoc-openapi-data-rest:1.6.11")
   implementation("org.springframework:spring-jms")
-  implementation("com.google.code.gson:gson:2.9.0")
-  implementation("uk.gov.justice.service.hmpps:hmpps-sqs-spring-boot-starter:1.1.7-beta-2")
+  implementation("com.google.code.gson:gson:2.9.1")
+  implementation("uk.gov.justice.service.hmpps:hmpps-sqs-spring-boot-starter:1.1.8")
 
-  runtimeOnly("com.h2database:h2:2.1.210")
-  runtimeOnly("org.flywaydb:flyway-core:8.5.1")
-  runtimeOnly("org.postgresql:postgresql:42.4.1")
+  runtimeOnly("com.h2database:h2:2.1.214")
+  runtimeOnly("org.flywaydb:flyway-core")
+  runtimeOnly("org.postgresql:postgresql:42.5.0")
 
-  testImplementation("org.flywaydb:flyway-core:8.5.1")
+  testImplementation("org.flywaydb:flyway-core")
```

Comment:
remove pinning which means that it will float to version specified by spring boot.  This avoids any incompatibility with newer versions of flyway